README.md

```diff
@@ -125,6 +125,32 @@ The following Environment Variables must be set in your shell prior to running a
 
 ---
 
+## Developer: Using the locally compiled Azure Provider binary
+
+After successfully compiling the Azure Provider, you must [instruct Terraform to use your locally compiled provider binary](https://www.terraform.io/docs/commands/cli-config.html#development-overrides-for-provider-developers) instead of the official binary from the Terraform Registy.
```

Comment:
This is only applicable to 0.14 and later, so this likely needs to be:

```suggestion
When using Terraform 0.14 and later, after successfully compiling the Azure Provider, you must [instruct Terraform to use your locally compiled provider binary](https://www.terraform.io/docs/commands/cli-config.html#development-overrides-for-provider-developers) instead of the official binary from the Terraform Registry.
```